frappe/utils/__init__.py

```diff
@@ -156,6 +155,25 @@ def split_emails(txt):
 
 	return email_list
 
+def validate_url(txt, throw=False):
```

Comment:
This will now replace the other `validate_url` function that was removed, which was meant for oauth operations

Reply:
This has been mentioned above already.

Reply:
What's the special case for oauth?

Reply:
Ah, my bad, I didn't see that comment. It's just a URL validation for OAuth requests.